testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/RPInitiatedLogoutTest.java

```diff
@@ -627,6 +627,36 @@ public void testFrontChannelLogoutWithPostLogoutRedirectUri() throws Exception {
         }
     }
 
+    @Test
+    public void testFrontChannelLogoutWithoutSessionRequired() throws Exception {
```

Comment:
It seems to be better to add a test where frontchannel_logout_session_required is false.

Reply:
I am not sure I understand this comment?

- The `testFrontChannelLogoutWithPostLogoutRedirectUri` tests the case when  frontchannel_logout_session_required is true
- The `testFrontChannelLogoutWithoutSessionRequired` tests the case when  frontchannel_logout_session_required is false

So both cases are covered.

Or maybe I misunderstood what exactly you meant by this comment?

Reply:
Sorry, I've misunderstood. Your original codes are fine.